test/README

```diff
@@ -103,15 +109,16 @@ A test machine image is created with vm-create, like so:
 
   $ ./vm-create -v -f cockpit
 
-The image will be created in $TEST_DATA/images/.  The vm-create
-command will not overwrite existing test machine images and simply
-report success in that case.  Use the "--force" option to create a new
-image no matter what.
+The image will be created in $TEST_DATA/images/. In addition a link
+reference will be created.
 
-The vm-create command will try to download an existing image from
-file.cockpit-project.org instead of creating a new image.
+If you wish that others use this new image then you should commit the
+new reference link, and use `vm-upload` to upload the new image. You
+would need to have Cockpit commit access to do this:
 
-There can be more than one test machine image.  For example, you might
+  $ ./vm-upload cockpit-fedora-22-x86_64
+
+There can be more than one test machine image. For example, you might
```

Comment:
What does this mean exactly? We definitely use more than one test machine image, so I'm not sure why we use "can be" here.